lib/core/src/server/templates/base-preview-head.html

```diff
@@ -5,6 +5,10 @@
     width: 100%;
     height: 100%;
   }
+  
+  body {
+    width: calc(100% - 16px);
```

Comment:
what is this magic 16px number?

Reply:
2x 8px margin

Reply:
But people may have other margin values

Reply:
this is the margin hardcoded by storybook in the iframe, not from the user.

Reply:
Not by storybook, by browsers. But users can override it

Reply:
Yeah this seems fragile to me @shilman